UNRELEASED.md

```diff
@@ -1,2 +1,6 @@
 # Unreleased
 
+**Breaking:**
+
+- Backpack/Font
+  - XXXL and XXL font sizes are now 36pt and 30pt (previously 42pt and 36pt). The 42pt font size has been removed for the time being.
```

Comment:
Would this make more sense the other way round? ie
```
XXL and XXXL font sizes are now 30pt and 36pt (previously 36pt and 42pt). The ???? font size has been removed for the time being.
```

Also I think you've got copy-pasta on the 42pt size